web/pgadmin/static/js/SchemaView/utils.js

```diff
@@ -0,0 +1,167 @@
+/////////////////////////////////////////////////////////////
```

Comment:
I feel it should be rename to `common.js` since it has more common codes than utils.

Reply:
Done.